aim/web/ui_v2/src/pages/Metrics/components/TableColumns/TableColumns.tsx

```diff
@@ -52,7 +54,14 @@ function getTableColumns(): ITableColumn[] {
       title: 'Iteration',
       width: 150,
     },
-  ];
+  ].concat(
+    paramColumns.map((param) => ({
+      dataKey: param.replaceAll('.', '_'),
```

Comment:
why are dots replaced here?

Reply:
There was a problem with getting value by nested path inside `BaseTable` component, so as a quick fix I decided to pass flatten object as row data

Reply:
The issue is actually resolved with the latest commit